test/unit/lua_tweaks.c

```diff
@@ -82,12 +90,6 @@ test_newindex(void)
 	   strcmp(diag_last_error(diag_get())->errmsg,
 		  "Value must be boolean, number, or string") == 0,
 	   "check error");
-	is(luaT_dostring(L, "tweaks.bool_var = 999999999999999999LLU"), -1,
-	   "invalid value - big int");
-	ok(!diag_is_empty(diag_get()) &&
-	   strcmp(diag_last_error(diag_get())->errmsg,
-		  "Value must be boolean, number, or string") == 0,
-	   "check error");
```

Comment:
What is wrong with this test case?

Reply:
This test case covers error conditions checked by `luaT_tweaks_newindex`. Before this change `int64_t` was unsupported so `luaT_tweaks_newindex` raised an error. Now, it doesn't raise an error in this case. The tweak handler (`tweak_set_bool`) will still raise an error (a different one though) but this test case isn't about it. The latter is tested in the `test_bool_var` test case. So I removed this particular check because it isn't relevant to `luaT_tweaks_newindex` anymore.